package.json

```diff
@@ -20,6 +20,7 @@
     "express-ftwebservice": "^2.0.2",
     "express-handlebars": "^2.0.1",
     "fetch-er": "0.0.10",
+    "ftlabs-screens-carousel": "github:ftlabs/screens-carousel#production",
```

Comment:
Could we pull from a tagged commit rather than a branch name? With a branch name, any updates to that branch would be pulled into this project on it's next deployment